.rubocop_todo.yml

```diff
@@ -1,6 +1,6 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config --auto-gen-only-exclude --exclude-limit 100`
-# on 2025-03-11 17:22:00 UTC using RuboCop version 1.73.2.
+# on 2025-03-13 13:41:46 UTC using RuboCop version 1.74.0.
```

Comment:
After a rebase, you shouldn't have any changes like this...

Reply:
Ah, true...

Reply:
Because in master the last run of RuboCop was already with 1.74.0. See: https://github.com/openSUSE/open-build-service/pull/17436/files#diff-0e154e6de9b468b91b4e5069a97429580fc41751e8fb057811017951d4753bf7R3

Reply:
Ah, I know, you are linking `src/api/.rubocop_todo.yml` and the file we're discussing over is `.rubocop_todo.yml`. I'll rebase again though...